spec/models/package_manager/go_spec.rb

```diff
@@ -1,42 +1,58 @@
-require 'rails_helper'
+# frozen_string_literal: true
+
+require "rails_helper"
 
 describe PackageManager::Go do
-  let(:project) { create(:project, name: 'foo', platform: described_class.name) }
+  let(:project) { create(:project, name: "foo", platform: described_class.name) }
 
   it 'has formatted name of "Go"' do
-    expect(described_class.formatted_name).to eq('Go')
+    expect(described_class.formatted_name).to eq("Go")
   end
 
-  describe '#package_link' do
-    it 'returns a link to project website' do
+  describe "#package_link" do
+    it "returns a link to project website" do
       expect(described_class.package_link(project)).to eq("https://pkg.go.dev/foo")
     end
 
-    it 'ignores version' do
-      expect(described_class.package_link(project, '2.0.0')).to eq("https://pkg.go.dev/foo@2.0.0")
+    it "ignores version" do
+      expect(described_class.package_link(project, "2.0.0")).to eq("https://pkg.go.dev/foo@2.0.0")
     end
   end
 
-  describe '#documentation_url' do
-    it 'returns a link to project website' do
-      expect(described_class.documentation_url('foo', '2.0.0')).to eq("https://pkg.go.dev/foo@2.0.0?tab=doc")
+  describe "#documentation_url" do
+    it "returns a link to project website" do
+      expect(described_class.documentation_url("foo", "2.0.0")).to eq("https://pkg.go.dev/foo@2.0.0#section-documentation")
     end
   end
 
-  describe '#install_instructions' do
-    it 'returns a command to install the project' do
+  describe "#install_instructions" do
+    it "returns a command to install the project" do
       expect(described_class.install_instructions(project)).to eq("go get foo")
     end
 
-    it 'ignores version' do
-      expect(described_class.install_instructions(project, '2.0.0')).to eq("go get foo")
+    it "ignores version" do
+      expect(described_class.install_instructions(project, "2.0.0")).to eq("go get foo")
+    end
+  end
+
+  describe "#get_repository_url" do
+    it "follows redirects to get correct url" do
+      VCR.use_cassette("go_redirects") do
+        expect(described_class.get_repository_url({ "Package" => "github.com/DarthSim/imgproxy" })).to eq("https://github.com/imgproxy/imgproxy")
+      end
     end
   end
 
-  describe '#get_repository_url' do
-    it 'follows redirects to get correct url' do
-      VCR.use_cassette('go_redirects') do
-        expect(described_class.get_repository_url({'Package' => 'github.com/DarthSim/imgproxy'})).to eq("https://github.com/imgproxy/imgproxy")
+  describe "#mapping" do
```

Comment:
This is the only thing changed in here. I added this test to confirm we pull in the data from the HTML response. The rest of the changes are rubocop.